x/programs/examples/token.go

```diff
@@ -124,18 +135,18 @@ func (t *Token) Run(ctx context.Context) error {
 	)
 
 	// transfer 50 from alice to bob
-	transferToBob := uint64(50)
-	_, err = runtime.Call(ctx, "transfer", contractId, alicePtr, bobPtr, transferToBob)
+	transferToBob := int64(50)
```

Comment:
Is there a reason why this is `int64` now instead of `uint64`?

Reply:
add todo and fixed on next push